website/docs/r/logs_custom_pipeline.html.markdown

```diff
@@ -89,6 +89,16 @@ resource "datadog_logs_custom_pipeline" "sample_pipeline" {
             is_enabled = true
         }
     }
+    processor {
+        lookup_processor {
+            source = "service_id"
+            target = "service_name"
+            lookup_table = ["1", "my service"]
```

Comment:
Did you want to map `1` to `my service`?
```suggestion
            lookup_table = ["1,my service"]
```

Reply:
Indeed, nice catch! It is fixed now.